chore(installation_docs): updated Linux installation guide in docker.md (#4593)
CircuitVerse/CircuitVerse: installation_docs/docker.md

|  Name | Version | Installation |
| --- | --- | --- |
| Git | Latest | [Download & Install](https://git-scm.com/downloads) |
| Docker Engine | Latest | [Follow this guide to install](https://docs.docker.com/engine/install/)  ⚠️ Don't go for `Docker Desktop`. Check the `Supported Platforms` table to install docker engine CLI for your operating system |

**Note**

Before installing Docker Engine, ensure that Docker Desktop is not already installed on your local machine. If Docker Desktop is installed, you can follow this [guide](https://docs.docker.com/desktop/uninstall/) to completely remove it from your Linux machine.

> After removing Docker Desktop, proceed with the installation of Docker Engine by following the steps outlined in the '**Install using the apt repository**' section on the [official Docker website](https://docs.docker.com/engine/install/ubuntu/).

Reviewer: All linux distro doesn't have apt package manager, so removing that
```suggestion
After removing Docker Desktop, proceed with the installation of Docker Engine as specified in the docs.
```
